docs/guides/developer/docs/api/workflow-api.md

```diff
@@ -0,0 +1,587 @@
+
+`400 (BAD REQUEST)`: The request is invalid or inconsistent.
+
+__Example__
+
+```
+[
+  {
+    "workflow_definition_identifier": "fast",
+    "parent_identifier": null,
+    "identifier": 1603,
+    "creator": "Opencast Project Administrator",
+    "operations": [
+      {
+        "identifier": 1604,
+        "configuration": {
+          "publishLive": "false",
+          "uploadedSearchPreview": "false",
+          "publishToOaiPmh": "true",
+          "comment": "false",
+          "publishToMediaModule": "true"
+        },
+        "time_in_queue": 0,
+        "failed_attempts": 0,
+        "start": "2018-08-08T08:46:57.3Z",
+        "description": "Applying default configuration values",
+        "fail_workflow_on_error": true,
+        "unless": "",
+        "max_attempts": 1,
+        "host": "http:\/\/localhost:8080",
+        "state": "succeeded",
+        "operation": "defaults",
+        "if": "",
+        "retry_strategy": "none",
+        "exception_handler_workflow": ""
+      },
+      {
+        "identifier": 1605,
+        "configuration": {
+          "apply-acl": "true"
+        },
+        "time_in_queue": 0,
+        "failed_attempts": 0,
+        "start": "",
+        "description": "Applying access control entries from series",
+        "fail_workflow_on_error": true,
+        "unless": "",
+        "max_attempts": 1,
+        "host": "http:\/\/localhost:8080",
+        "state": "skipped",
+        "operation": "series",
+        "if": "",
+        "retry_strategy": "none",
+        "exception_handler_workflow": "partial-error"
+      }
+      ...
+    ],
+    "configuration": {
+      "publishLive": "false",
+      "workflowDefinitionId": "fast",
+      "uploadedSearchPreview": "false",
+      "publishToOaiPmh": "true",
+      "comment": "false",
+      "publishToMediaModule": "true"
+    },
+    "description": "\n    A minimal workflow that transcodes the media into distribution formats, then\n    sends the resulting distribution files, along with their associated metadata,\n    to the distribution channels.\n  ",
+    "state": "succeeded",
+    "title": "Fast Testing Workflow",
+    "event_identifier": "f41e4417-b841-4d20-a466-f98ddfbe4c2a"
+  }
+]
+```
+
+### POST /api/workflows
+
+Creates a workflow instance.
+
+Form Parameters                  | Required |Type                             | Description
+:--------------------------------|:---------|:--------------------------------|:-----------
+`event_identifier`               | yes      | [`string`](types.md#basic)      | The event identifier this workflow should run against
+`workflow_definition_identifier` | yes      | [`string`](types.md#basic)      | The identifier of the workflow definition to use
+`parent_identifier`              | no       | [`integer`](types.md#basic)     | The optional parent workflow identifier
```

Comment:
This seems quite "internal" to me. Is this required in the External API?

Reply:
When I implemented this, I checked whether the parent ID is used at all and the only place where it was set is in [WorkflowRestService.java](https://github.com/opencast/opencast/blob/1871a85d8a1b3cc51fea714c1fecc49aabbf377a/modules/workflow-service-impl/src/main/java/org/opencastproject/workflow/endpoint/WorkflowRestService.java#L566-L619). What is a parent/child workflow anyway? So I figured that maybe some adopters are using this or it is dead code. Should I remove it then?

Reply:
I would recommend to not include it in the External API. If you have the time, it would make sense to remove it completely (if unused), but that is not in the scope of this PR.